service.js

```diff
@@ -40,14 +41,26 @@ seneca.options.sanitizeTextArea = {
     img: ['*']
   })
 };
-seneca.decorate('customValidatorLogFormatter', require('./lib/custom-validator-log-formatter'));
 seneca.use(store, config['postgresql-store']);
 seneca.use(storeQuery);
+seneca.decorate('customValidatorLogFormatter', require('./lib/custom-validator-log-formatter'));
 seneca.use(require('./lib/cd-events'), { logger: log.logger });
 seneca.use(require('cp-permissions-plugin'), {
-  config: __dirname + '/config/permissions'
+  config: `${__dirname}/config/permissions`
 });
 
+if (!_.isUndefined(newrelic)) {
```

Comment:
isUndefined is a bit of an overkill here as undefined is falsy

Reply:
/shrug 
better safe than sorry